mlir/lib/Conversion/AIRLoweringPass.cpp

```diff
@@ -1300,8 +1300,10 @@ class AIRLoweringPass : public air::AIRLoweringBase<AIRLoweringPass> {
         merge_candidate_loop = true;
       else if (auto scf_par = dyn_cast<scf::ParallelOp>(scf_loop)) {
         merge_candidate_loop = false;
-        for (auto child_scf_for_loop : scf_par.getBody()->getOps<scf::ForOp>())
+        for (auto child_scf_for_loop : scf_par.getBody()->getOps<scf::ForOp>()) {
```

Comment:
Just curious about what this code does.
I guess it is not just testing if `scf_par.getBody()->getOps<scf::ForOp>()` is non empty?

Reply:
yeah, there's a bound of nonsense code here.